src/pru/Makefile

```diff
@@ -27,7 +27,11 @@ target-y += $(OUT)pru0.elf $(OUT)pru1.elf
 $(OUT)pru0.elf: $(patsubst %.c, $(OUT)src/%.o,$(pru0-y))
 	@echo "  Linking $@"
 	$(Q)$(CC) $(CFLAGS_klipper.elf) $^ -o $(OUT)pru0.o
+	# dumping info about INTC
+	$(Q)$(OBJCOPY) --dump-section '.pru_irq_map'=$(OUT)pru0.pru_irq_map.bin $(OUT)src/pru/pru0.o $(OUT)pru0.o.discard
 	$(Q)$(CC) $(CFLAGS_pru0.elf) $(OUT)pru0.o -o $@
+	# in previous step linker did loose info about INTC during optimization, restoring it here (P.S. pru.lds is not helping)
+	$(Q)$(OBJCOPY) --add-section '.pru_irq_map'=$(OUT)pru0.pru_irq_map.bin $@
```

Comment:
Can you describe what this does?  Could modifying the `src/pru/pru.lds` linker script accomplish the same goal instead?

Reply:
Believe me I did try everything what i know  to fix this issue by changing pru.lds - but i was still losing ".pru_irq_map" - so i made this workaround.